sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BatchLoads.java

```diff
@@ -372,7 +372,7 @@ public WriteResult expandUntriggered(PCollection<KV<DestinationT, TableRow>> inp
 
     tempTables
         .apply("ReifyRenameInput", new ReifyAsIterable<>())
-        .setCoder(IterableCoder.of(KvCoder.of(TableDestinationCoderV2.of(), StringUtf8Coder.of())))
+        .setCoder(IterableCoder.of(KvCoder.of(TableDestinationCoderV3.of(), StringUtf8Coder.of())))
```

Comment:
Same here. The coded value is only passed to WriteRename which creates a copy job and never references the clustering configuration.